diff --git a/node_modules/react-native-sound/android/build.gradle b/node_modules/react-native-sound/android/build.gradle
index de375bf..5accd02 100644
--- a/node_modules/react-native-sound/android/build.gradle
+++ b/node_modules/react-native-sound/android/build.gradle
@@ -1,24 +1,13 @@
 apply plugin: 'com.android.library'
 
-def DEFAULT_COMPILE_SDK_VERSION             = 26
-def DEFAULT_BUILD_TOOLS_VERSION             = "26.0.2"
-def DEFAULT_TARGET_SDK_VERSION              = 26
-
 android {
-    compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
-    buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
-
-  defaultConfig {
-    minSdkVersion 16
-    targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : DEFAULT_TARGET_SDK_VERSION
-    versionCode 1
-    versionName "1.0"
-    ndk {
-      abiFilters "armeabi-v7a", "x86"
+    namespace "com.zmxv.RNSound"
+    compileSdk rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : 35
+    defaultConfig {
+        minSdkVersion rootProject.hasProperty('minSdkVersion') ? rootProject.minSdkVersion : 24
     }
-  }
 }
 
 dependencies {
-  compileOnly 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-android'
 }
diff --git a/node_modules/react-native-sound/android/src/main/AndroidManifest.xml b/node_modules/react-native-sound/android/src/main/AndroidManifest.xml
index d961e60..babb201 100644
--- a/node_modules/react-native-sound/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-sound/android/src/main/AndroidManifest.xml
@@ -1,3 +1,2 @@
-<manifest xml:android="http://schemas.android.com/apk/res/android"
-          package="com.zmxv.RNSound">
+<manifest xml:android="http://schemas.android.com/apk/res/android">
 </manifest>
